### PR TITLE
Added traduction for pt-br of the additional-methods-and-properties page

### PR DESCRIPTION
**Pull request by clarel**
Almost finished the traduction of the page, just leave some of technical terms that is hard to translate in that context.

### CODE REVIEW

File: pt/models/additional-methods-and-properties.rst [hunk 1]
```diff
@@ -1,17 +1,109 @@
-Additional Methods and Properties
+Métodos e Propriedades Adicionais (Additional Methods and Properties)
 #################################
```
Comment (cake17) [inline pt/models/additional-methods-and-properties.rst]: the line with # should be as long as the title

**Comment by cake17** [comment]
thanks @clarel nice work :+1: 
We tried to keep the most english terms in french translation (not that easy sometimes to make sentences understandable...)

**Comment by clarel** [comment]
i see, i will keep translating while i'm studying the cakephp!

**Comment by cake17** [comment]
thanks for your help :smile:

**Pull request merged**